lib/profiling/lfric/constants_mod.F90

```diff
@@ -0,0 +1,154 @@
+!-----------------------------------------------------------------------------
+! Copyright (c) 2017,  Met Office, on behalf of HMSO and Queen's Printer
+! For further details please refer to the file LICENCE.original which you
+! should have received as part of this distribution.
```

Comment:
This needs to be modified, e.g. examples/lfric/eg4/mm_diagonal_kernel_mod.F90

Reply:
Since we've modified it, it also then gains the 3-clause BSD license for the *modifications*. Again, see that file I've just pointed to.

Reply:
All files are now used unmodified from the infrastructure directory (only additional modification was to mpi_mod, which was already modified).